external_projects/bccm_player/android/src/main/kotlin/media/bcc/bccm_player/players/exoplayer/ExoPlayerController.kt

```diff
@@ -186,6 +190,35 @@ class ExoPlayerController(private val context: Context) :
         }
     }
 
+    /**
+     * ExoPlayers `setPreferredTextLanguage` makes it impossible to
+     * disable the subtitle.
+     *
+     * Instead of `setPreferredLanguage` we force the text track to default languag (from settings)
+     * once. This method handles this behavior.
+     */
+    private fun setDefaultTextTrack(tracks: Tracks) {
+        if (textLanguageThatShouldBeSelected == null) {
```

Comment:
Suggestion
- rename this to setTextTrack
- move the "run only once"-logic into `onTracksChanged`

Reply:
would make it more readable whats going on